sa/sa.go

```diff
@@ -869,6 +870,67 @@ func (ssa *SQLStorageAuthority) RevokeAuthorizationsByDomain(ctx context.Context
 	return results[0], results[1], nil
 }
 
+// RevokeAuthorizationsByDomain2 invalidates all pending or valid authorizations for a
```

Comment:
90% of stuff that uses the new methods don't actually use them yet, the only places that do were the WFE/RA for Get/New methods because that's all this PR was originally supposed to be. There are follow-up issues for most of the remaining wiring work but I'll stick them all in a project or something for more visibility.